### PR TITLE
feat(hlapi): add 32, 64, 128 bits types

### PR DESCRIPTION
**Pull request by tmontaigu**

### CODE REVIEW

File: tfhe/src/typed_api/integers/types/base.rs [hunk 1]
```diff
@@ -68,25 +68,38 @@ where
...
 {
-    fn decrypt(&self, key: &ClientKey) -> u64 {
+    fn decrypt(&self, key: &ClientKey) -> u16 {
         let key = self.id.unwrapped_ref_key(key);
         key.inner.decrypt(&self.ciphertext.borrow())
     }
 }
 
-impl<P> FheDecrypt<U256> for GenericInteger<P>
+impl<P> FheDecrypt<u32> for GenericInteger<P>
```
Comment (IceTDrinker) [inline tfhe/src/typed_api/integers/types/base.rs:83]: u256 and u64 still have their impls somewhere ?

**Review by IceTDrinker — APPROVED** [review]
Looks good small questions/remarks but otherwise ok !

**Comment by IceTDrinker** [comment]
otherwise I would not approved 😅

**Comment by tmontaigu** [comment]
It was also a way to re-trigger the CI without having to type in the commands 🤪

**Pull request merged**